server/__init__.py

```diff
@@ -1,14 +1,14 @@
 import logging
 import sys
-from backend.common.utils.utils import import_plugins
+from server.common.utils.utils import import_plugins
 
 __version__ = "0.16.7"
```

Comment:
Can we update this to pull from the current commit hash?

Reply:
Not sure what you mean. Will follow up